headers/validation.h

```diff
@@ -166,11 +213,25 @@ struct UniformGenerator {
 	template <typename T>
 	T operator()(T low, T high, std::mt19937_64& rng) const {
 		static_assert(is_number_v<T>);
-		if constexpr(std::is_same_v<T, long long>)
-			return std::uniform_int_distribution<T>(low, high)(rng);
```

Comment:
It would be cool if we could somehow ban the distributions from this file. Since `<random>` is also needed for RNGs, that doesn't seem feasible though.

Reply:
maybe with some macro magic... but i don't know if that's possible... alternatively BAPCtools could check files for this as with `#include <bits/stdc++.h>`?

Reply:
> alternatively BAPCtools could check files for this as with #include <bits/stdc++.h>?

yeah, I already check for `list(set(..))` in python as well since sets don't have very stable ordering. Could add the distrubitions to that list. But lets do that in a separate PR.